packages/swingset-runner/demo/vatStore2/thingHolder.js

```diff
@@ -1,7 +1,9 @@
-/* global makeKind makeVirtualScalarWeakMap */
+/* global VatData */
```

Comment:
Verbally agreed today that we don't. But we shouldn't use it until this is somehow fixed. I like @michaelfig 's suggestion of a module that just reads these globals and then exports them. Everyone else can then just import them from that module.

Even better would be if that special module could also delete them from the global after reading them into its own state.

Reply:
Creating such a module seems straightforward, but poses a couple of dangling questions for me:

(1) I'm presuming we handle the case of running outside a vat (as in tests) by having the mock implementation set `VatData` (as it does), so you could just do your mock setup at the top of tests the way you do now and from there anything that imports the aforementioned module would pick up the mock `VatData`.  Do I have that right?

(2) Where does this module live and what is it called?  The obvious thing would be `@agoric/store` but that already is used by the `store` package to export the stuff _it_ exports, much of which should not be used outside agoricSDK anyway.  Indeed, this new module we're talking about should probably also grab the public part of that (such as `M` and the ephemeral store makers) for reexport so that the new module can be the one-stop shop that user code imports for all its store needs.